cmd/networkcmd/clean.go

```diff
@@ -59,11 +50,9 @@ func clean(*cobra.Command, []string) error {
 		return err
 	}
 
-	if hard {
-		ux.Logger.PrintToUser("hard clean requested via flag, removing all downloaded avalanchego and plugin binaries")
-		binDir := filepath.Join(app.GetBaseDir(), constants.AvalancheCliBinDir)
-		cleanBins(binDir)
-	}
+	ux.Logger.PrintToUser("removing all downloaded avalanchego and plugin binaries")
+	binDir := filepath.Join(app.GetBaseDir(), constants.AvalancheCliBinDir)
+	cleanBins(binDir)
```

Comment:
I mean, we don't even need this functionality just don;t do it. Lets keep the downloaded binaries after a clean.

Reply:
removed this block